netbench/netbench-cli/src/report.rs

```diff
@@ -391,6 +414,7 @@ stat!(
         ContextSwitches = "context-switches",
         Syscalls = "syscalls",
         Connections = "connections",
+        ConnectTime = "connect-time",
```

Comment:
the graph shows the y axis is `m`. I thought we were measuring this in seconds? can you add units to also maybe. `connect-time(seconds)`

Reply:
or maybe the more appropriate question is, how do you read the connect-time graph?

Reply:
it's average connect time for all handshakes performed in that second window